benchmark/runner/mini_runners.cpp

```diff
@@ -681,42 +681,82 @@ static void GenUpdateDeleteIndexArguments(benchmark::internal::Benchmark *b) {
 /**
  * Arg <0, 1, 2, 3, 4, 5>
  * 0 - # integers to scan
- * 1 - # bigints to scan
+ * 1 - # mixed type to scan
  * 2 - # integers in table
- * 3 - # bigints in table
+ * 3 - # mixed type in table
  * 4 - row
  * 5 - cardinality
+ * 6 - mixed type is a varchar or not
+ * 7 - # possible threads
  */
 static void GenCreateIndexArguments(benchmark::internal::Benchmark *b) {
-  auto num_cols = {1, 3, 5, 7, 9, 11, 13, 15};
+  // 0 is a special argument used to indicate serial
+  auto num_threads = {0, 1, 2, 4, 8, 16};
+  auto num_cols = {1, 2, 3, 4, 5, 7, 9, 11, 13, 15};
```

Comment:
Similar here. Extract this into a common global vector. Also, define the column combination for the mini runner tables in this file and pass them into `GenerateMiniRunnerTables` as argument. In general, remove hard-coded values in `table_generator.cpp` (also applies to `GenerateMiniRunnerIndexTables`). Put all the mini-runner related parameters in this file.